zerver/tests/test_webhooks_common.py

```diff
@@ -140,6 +144,47 @@ def test_standardize_headers(self) -> None:
         expected_djangoified_headers = {"CONTENT_TYPE": "text/plain", "HTTP_X_EVENT_TYPE": "ping"}
         self.assertEqual(djangoified_headers, expected_djangoified_headers)
 
+    def test_validate_webhook_signature(self) -> None:
+        with self.settings(VERIFY_WEBHOOK_SIGNATURES=True):
```

Comment:
You can make use the `@override_settings` decorator and save some indentation of the whole test.